install.py

```
@@ -0,0 +1,349 @@
+            # Lade Ollama-Installationsskript
+            subprocess.check_call([
+                "curl", "-fsSL", "https://ollama.ai/install.sh", "|", "sh"
+            ], shell=True)
+            print("✅ Ollama erfolgreich installiert")
+        except:
+            print("⚠️  Manuelle Installation erforderlich: https://ollama.ai/download/linux")
+    
+    # Lade Standard-Modell
+    print("\n🧠 Lade KI-Modell (llama2:7b)...")
+    try:
+        subprocess.check_call(["ollama", "pull", "llama2:7b"])
```

Ruff (0.12.2)

Starting a process with a partial executable path

(S607)